ec2removeimg

```diff
@@ -164,7 +164,7 @@ if (
     sys.exit(1)
 
 
-config_file = args.config_file_path
+config_file = os.path.expanduser('~') + os.sep + args.config_file_path
```

Comment:
And here expand the entire value:

```
config_file = os.path.expanduser(args.config_file_path)
```

Otherwise the user can't use the ~ in path and can't provide any path that's outside of home directory.